src/veritysetup/veritysetup.c

```diff
@@ -60,19 +61,22 @@ static int parse_flags(const char *flags) {
                 if (r == 0)
                         break;
 
+                if (STR_IN_SET(word, "noauto", "auto", "nofail", "fail"))
```

Comment:
Please split out changes to this file, apart from this one if block, from commit "veritysetup-generator: add support for veritytab" and squash it back into the first commit, "veritysetup: add support for dm-verity flags"

We want "perfect" commits in the git history